api/v1beta2/reference.go

```diff
@@ -16,7 +16,11 @@ limitations under the License.
 
 package v1beta2
 
-import "fmt"
+import (
+	"fmt"
+
+	imagev1_reflect "github.com/fluxcd/image-reflector-controller/api/v1beta2"
```

Comment:
API packages shouldn’t import each other. Please undo this.

Reply:
Fixed, thanks!

Reply:
Looks like we need to remove this reference as well at some point:

https://github.com/fluxcd/image-automation-controller/blob/6c5ffee6c00d92f43e7e9d9af37a9e96310b6e0e/api/v1beta2/git.go#L21